app/api/applications.js

```diff
@@ -28,36 +25,15 @@ async function getApplications (searchType, searchText, limit, offset, filterSta
     json: true
   }
   try {
-    const response = await Wreck.post(url, options)
-    if (response.res.statusCode !== 200) {
-      return { applications: [], total: 0, applicationStatus: [] }
-    }
-    return response.payload
+    const { payload } = await wreck.post(endpoint, options)
+    return payload
   } catch (err) {
-    console.log(err)
-    return { applications: [], total: 0, applicationStatus: [] }
+    logger.setBindings({ err, endpoint })
+    throw err
   }
 }
 
-async function withdrawApplication (reference, user, status) {
```

Comment:
👏 Removing unused stuff, might be a first for this application, but good to see!